internal/chain/reward.go

```diff
@@ -45,34 +46,35 @@ func ballotResultBeaconchain(
 }
 
 var (
-	votingPowerCache   singleflight.Group
-	delegateShareCache singleflight.Group
+	votingPowerCache, _   = lru.New(16)
```

Comment:
why 16 here?  can you elaborate it a bit?

Reply:
votingPower is the calcuation per committee (or per epoch/shard). so 16 is good enough for multiple epochs.